<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullMainWindow</class>
 <widget class="QMainWindow" name="FullMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>32</number>
    </property>
    <property name="topMargin">
     <number>48</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>155</number>
      </property>
      <property name="rightMargin">
       <number>35</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>372</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>372</width>
          <height>36</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="fullSelectToolButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/data/img/mainviewwidget/全屏 icon-功能排序.png</normaloff>:/data/img/mainviewwidget/全屏 icon-功能排序.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="fullSelectMenuButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/data/img/mainviewwidget/全屏 icon- 下拉.svg</normaloff>
          <normalon>:/data/img/mainviewwidget/全屏 icon- 下拉.svg</normalon>
          <activeon>:/data/img/mainviewwidget/全屏 icon- 下拉.png</activeon>:/data/img/mainviewwidget/全屏 icon- 下拉.svg</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minPushButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/data/img/mainviewwidget/全屏 icon-缩小.png</normaloff>:/data/img/mainviewwidget/全屏 icon-缩小.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="fullStackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="FullCommonUseWidget" name="fullCommonPage"/>
      <widget class="FullLetterWidget" name="fullLetterPage"/>
      <widget class="FullFunctionWidget" name="fullFunctionPage"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FullCommonUseWidget</class>
   <extends>QWidget</extends>
   <header>src/CommonUseWidget/fullcommonusewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FullFunctionWidget</class>
   <extends>QWidget</extends>
   <header>src/FunctionWidget/fullfunctionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FullLetterWidget</class>
   <extends>QWidget</extends>
   <header>src/LetterWidget/fullletterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
